src-docs/src/components/guide_page/guide_page_chrome.js

```diff
@@ -142,14 +141,17 @@ export class GuidePageChrome extends Component {
             responsive={false}
             wrap>
             <EuiFlexItem grow={false}>
-              <Link to="/" className="guideLogo" aria-label="Go to home page">
+              <EuiLink
+                to="/"
+                className="guideLogo"
+                aria-label="Go to home page">
                 <EuiIcon type="logoElastic" size="l" />
-              </Link>
+              </EuiLink>
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
-              <Link to="/" aria-label="Go to home page" className="euiLink">
+              <EuiLink to="/" aria-label="Go to home page" className="euiLink">
```

Comment:
Nit: Same as above

```suggestion
              <EuiLink to="/" aria-label="EUI Home" className="euiLink">
```
Meta-nit: These should probably one `aria-label` value set to a variable so the strings don't by accident float apart?

Meta-nit: kind of annoying that we have two links that go to the same spot but I'm guessing that a link with an icon defined doesn't give the right sizing for it?

Reply:
> Meta-nit: kind of annoying that we have two links that go to the same spot but I'm guessing that a link with an icon defined doesn't give the right sizing for it?

Naively combining the two ends up as

<img width="223" alt="Screen Shot 2020-10-16 at 11 01 34 AM" src="https://user-images.githubusercontent.com/313125/96287367-0ef3af80-0f9f-11eb-9f5d-6fe0910f8b38.png">

Reply:
Turns out we have a design pattern & components for this 😊  - pushed!